src/pages/about.js

```diff
@@ -0,0 +1,51 @@
+import { graphql } from "gatsby"
+
+import React from "react"
+
+export default function About({ data }) {
+    const aboutData = data.prismicAboutPage.data
+
+    return (
+        <div>
+            <h1>{aboutData.company_background_section_heading}</h1>
```

Comment:
Same thing as on #4 , please just utilize all of the pulled data so you can show the client.